gammapy/datasets/map.py

```diff
@@ -2572,8 +2575,16 @@ def _is_stackable(self):
     def stack(self, other, nan_to_num=True):
         r"""Stack another dataset in place.
 
-        The ``acceptance`` of the stacked dataset is normalized to 1,
-        and the stacked ``acceptance_off`` is scaled so that:
+        Safe mask is applied to the other dataset to compute the stacked counts data.
+        Counts outside the safe mask are lost.
+
+        The ``acceptance`` of the stacked dataset is obtained by stacking the acceptance weighted
+        by the other mask_safe onto the current unwieghted acceptance.
+
+        Note that the masking is not applied to the current dataset. If masking needs
+        to be applied to it, use `~gammapy.MapDataset.to_masked()` first.
+
+        The stacked ``acceptance_off`` is scaled so that:
```

Comment:
Maybe one can put below the formulae for both acceptance, adding the use of the masks (in the `MapDataset.stack' docstring, one uses \epsilon for the mask

Reply:
We could instead put that information here https://docs.gammapy.org/1.2/user-guide/makers/ring.html ? 
But, no strong opinion on if it should also be in the docstring

Reply:
As a user, I think it would definitely be helpful to have the formulae in the user guide (and maybe linked in the docs). Having it in the docstring might be nice too.